src/store/database/sagas.ts

```diff
@@ -76,7 +85,23 @@ export function* fetchDatabases(
   }
 }
 
+const POLL_DATABASE_COOLDOWN_MS = 5000;
+
+export function* pollDatabases(): Generator<any> {
+  while (true) {
+    yield take(getType(databasesSubscribe));
+    while (
+      yield select((state: AppState) => state.database.databasesSubscribed)
```

Comment:
Let's get into the habit of using `reselect` memoized selectors for all state queries. It's not totally important here, but we should build the patterns for future uses

Reply:
I'll follow up with a separate PR to refactor all selectors we have so far.